galite-core/src/main/kotlin/org/kopi/galite/visual/dsl/form/FormField.kt

```diff
@@ -400,8 +402,14 @@ open class FormField<T>(internal val block: Block,
   }
 
   fun setInfo(source: String, form: VForm) {
+    var source = source
+
+    if(!domain.javaClass.simpleName.isNullOrEmpty()) {
```

Comment:
When using JDK8, `domain.javaClass.simpleName` doesn't returning null when domain is an anonymous object.